mgmt_cli_test.py

```diff
@@ -185,6 +189,37 @@ def test_backup_multiple_ks_tables(self):
         self.verify_backup_success(bucket=location_list[0].split(':')[1], cluster_id=backup_task.cluster_id)
         self.log.info('finishing test_backup_multiple_ks_tables')
 
+    def test_backup_location_with_path(self):
+        self.log.info('starting test_backup_location_with_path')
+        self.update_cred_file()
+        location_list = ['s3:manager-backup-tests-us/path_testing/']
+        manager_tool = mgmt.get_scylla_manager_tool(manager_node=self.monitors.nodes[0])
+        mgr_cluster = manager_tool.add_cluster(name=self.CLUSTER_NAME + '_bucket_with_path',
+                                               db_cluster=self.db_cluster,
+                                               auth_token=self.monitors.mgmt_auth_token)
+        self._generate_load()
+        try:
+            mgr_cluster.create_backup_task({'location': location_list})
+        except ScyllaManagerError as error:
+            self.log.info(f'Expected to fail - error: {error}')
+        self.log.info('finishing test_backup_location_with_path')
+
+    def test_backup_rate_limit(self):
+        self.log.info('starting test_backup_rate_limit')
+        self.update_cred_file()
+        location_list = ['s3:manager-backup-tests-us']
+        manager_tool = mgmt.get_scylla_manager_tool(manager_node=self.monitors.nodes[0])
+        mgr_cluster = manager_tool.add_cluster(name=self.CLUSTER_NAME + '_rate_limit', db_cluster=self.db_cluster,
+                                               auth_token=self.monitors.mgmt_auth_token)
+        self._generate_load()
+        rate_limit = ','.join([f'{dc}:{randint(1, 10)}' for dc in self.get_all_dcs_names()])
+        self.log.info(f'rate limit will be {rate_limit}')
+        backup_task = mgr_cluster.create_backup_task({'location': location_list, 'rate-limit': rate_limit})
+        task_status = backup_task.wait_and_get_final_status()
+        self.log.info(f'backup task finished with status {task_status}')
```

Comment:
How does it really verifies that the rate limit works?
Need to add verification for that

Reply:
For it we must first to create a bigger data load, to have some time to verify the upload rate (it will use the existing functions that show the detailed task status, but it must be added after we create a specific load (even using C-S)

Reply:
ok, so create a big data load

Reply:
I'm ok to do it in a follow up PR.
Open yourself a task for it and when you have it, I can merge this one.

Reply:
created a [task](https://trello.com/c/xxrTYh0E/1179-manager-20)